json-schema-form.d.ts

```diff
@@ -114,20 +155,9 @@ export function createHeadlessForm(
   customConfig?: JSFConfig
 ): HeadlessFormOutput;
 
-type MutationsConfig = {
-  /**
-   * An object with the list of fields to be mutated.
-   */
-  fields: Record<string, unknown>;
-  /**
-   * a callback function that applies a mutation to all the fields.
-   * @param fieldName
-   * @param fieldAttrs
-   * @returns
-   */
-  allFields: (fieldName: string, fieldAttrs: Record<string, unknown>) => Record<string, unknown>;
-};
-
+/**
+ * Modifies the JSON schema based on the provided configuration.
+ */
 export function modify(
```

Comment:
bug: @joaoalmeida-remote I was trying the new version at Remote codebases when I noticed the declarations were outdated (and breaking my TS code). We cannot let that happen for future releases.
We need a way (test?) to automatically detect when the TS declarations are wrong. Related [Linear issue](https://linear.app/remote/issue/RMT-15/jsf-add-typescript)